### ISSUE TITLE
Error: Server Error when trying to open the pipeline

### ISSUE THREAD

**Issue by tkraeft** [google/crmint#232]
Hi, since august 1st I can't log in to my crmint account (dmch-255408). As soon as I open the path, I get redirected to https://oa.appengine.google.com/_ah/conflogin where I'm asked to grant access to my Google account. But then the following message appears:
"Error: Server Error
The server encountered an error and could not complete your request.
Please try again in 30 seconds." Since 10 days now.
There are multiple accounts with access to the interface, but none can access it. The processes are still running, as the data flows and I also receive notifactions about the success. 
Any help would be much appreciated!

**Comment by tyroneschiff** [comment]
I don't have a resolution for you, unfortunately, however, this same thing did happen to me today when deploying from the dev environment. The user who was trying to access the CRMint account could not get through. We worked around the problem by inviting another of his account's to the CRMint account. Strangely the other Google account could access the CRMint account, but the first could not.

**Comment by tkraeft** [comment]
Thanks a lot @tyroneschiff, that would be a nice idea. But is there a way to add users when not having access to the UI? All active accounts do get the same error message at the moment, so I can't log in to add users. The pipelines are by the way still running, so it's really just an access problem.

**Comment by tyroneschiff** [comment]
Access to the CRMint app is managed by the IAM & Admin section in Google Cloud. You can go directly to your GCP project, give credentials to a new account, and retry accessing the CRMint UI when logged in from that identity.

**Comment by tkraeft** [comment]
You are right @tyroneschiff this worked for the new account as well. Stil the existing ones don't get access although being project owners. Anyway thanks a lot for helping out here!

**Comment by dulacp** [comment]
We are closing this issue since the new release of CRMint 2.0 should fix this issue, please see the details in the mentioned above.
